api/specs.proto

```diff
@@ -370,6 +372,8 @@ message ContainerSpec {
 	// Ulimits defines the list of ulimits to set in the container. This option
 	// is equivalent to passing --ulimit to docker run.
 	repeated Ulimit ulimits = 29;
+
+	int64 oom_score_adj = 30;
```

Comment:
Please add a comment, in the fashion of the comments above, explaining what the field does. These end up as godoc comments, so the comment should start with something like

```go
// OomScoreAdj <sets/defines/verbs>
```